### ISSUE TITLE
Various lists on MyStuff page

### ISSUE THREAD

**Issue by wdehoog** [sailfish-spotify/hutspot#18]
Currently the MyStuff page contains five sets: saved albums, saved tracks, playlists, followed artists and recently played tracks. Should it remain this way, should the UI be changed or should they be on different pages?

**Comment by BonoNL** [comment]
The page seems a bit cluttered.
To much info, not all albums are shown and how to get them show?

Maybe an idea to collapse the list when pressing the header/title?

**Comment by abranson** [comment]
I suggest only one listing at a time, switching between them with a pulley menu.

**Comment by wdehoog** [comment]
You meant something like this?
![20180905165357](https://user-images.githubusercontent.com/3353303/45101897-c402df00-b12c-11e8-9658-871120e3baef.jpg)

**Comment by BonoNL** [comment]
Yes that is what I mean, now we need to swipe from on page to another.

I (personally) think that collapsing the list by pressing the letter could make this easy search able and keep a good overall view.

But that's just my opinion.
